src/pages/Tap.vue

```diff
@@ -77,11 +77,12 @@ export default {
                             response.data.invalid.length > 0
                         ) {
                             this.showFail();
-                            this.errorMessage = "[xXx]The voucher code you entered is not valid. Please try again.";
+                            this.errorMessage = "[xXx] Please enter a valid voucher code.";
                         } else if (
                             response.data.fail.length > 0
                         ) {
-                            this.errorMessage = "[xXx]The voucher code you entered has previously been submitted. Please try again.";
+                            this.showFail();
+                            this.errorMessage = "[xXx] Please enter an unused voucher code.";
```

Comment:
Makes me feel, as a user, like i didi this *on purpose* and am a bad person.
"That voucher may have been used already."